Makefile

```
@@ -254,6 +263,8 @@ build-ci: ci-embed-ui bin/ship
 
 build-ci-cypress: mark-ui-gitignored pkg/lifeycle/daemon/ui.bindatafs.go bin/ship
 
+build-minimal: bin/ship build-ui pkg/lifeycle/daemon/ui.bindatafs.go
```

```suggestion
build-minimal: build-ui pkg/lifecycle/daemon/ui.bindatafs.go bin/ship
```
As suggested by @dexhorthy 
Also, lifecycle was misspelled (and it is in `build-ci-cypress` too...)